app/controllers/content_items_controller.rb

```diff
@@ -31,24 +31,23 @@ def load_content_item
   end
 
   def present(content_item)
-    class_name = content_item["document_type"].sub(/^service_manual_/, "").classify
-    presenter_name = "#{class_name}Presenter"
-    presenter_class = Object.const_get(presenter_name)
-    presenter_class.new(content_item)
-  rescue NameError
     if content_item["base_path"] == "/"
```

Comment:
This took me a little while to grasp. Might be worth a comment to say explicitly that without this check we'll fetch the GOV.UK homepage content item and try to render it with the HomepagePresenter.

Reply:
A more robust check would ensure `service_manual_` was present before substituting it, but I guess we can rely on Router to protect the app in production. This is a net improvement and should be enough to avoid surprises in local development.

Reply:
That's a good idea, I've applied that suggestion.

Reply:
Nice 💯